index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>StoryStart</title>
+		<link type="text/css" rel="stylesheet" href="stylesheet.css"/>
+    </head>
+    <body>
+        
+		<form action="mailto:someone@example.com" method="post" enctype="text/plain">
+            
+        <div class="div-h">
+            <h1>StoryStart</h1>
+        </div>
+        <div class="div-p">
+            <div><p>NAME YOUR STORY<br/>

```

Comment:
For readability, tags such as the <p> usually either 1) vertically line up with the corresponding closing tag </p> or 2) are on the same line as the closing tag. In this case, the former would be appropriate.